### PR TITLE
pekko grpc 1.0.1 release

### PR DESCRIPTION
**Pull request by pjfanning**

### CODE REVIEW

File: project/plugins.sbt
Comment (mdedetrich) [inline project/plugins.sbt:39]: @pjfanning Also need to update it in `Dependencies.scala`

Comment (pjfanning) [reply project/plugins.sbt:39]: I can't find anything to update in Dependencies.scala.

This is all I can find.
```
  // Sync with plugins.sbt
  val PekkoGrpcBinaryVersion = "current"
```

Do you want `current` changed to `1.0.1`?

Comment (mdedetrich) [reply project/plugins.sbt:39]: > I can't find anything to update in Dependencies.scala.
> 
> This is all I can find.
> ```
>   // Sync with plugins.sbt
>   val PekkoGrpcBinaryVersion = "current"
> ```
> 
> Do you want `current` changed to `1.0.1`? 

Hmm that sounds off for the same reasons as pekko http where other caused issues. I am getting the nagging feeling that we should leave this to 1.0.0 and setup a system where we build against snapshots as a nightly job

Comment (pjfanning) [reply project/plugins.sbt:39]: Dependencies.PekkoGrpcBinaryVersion is only used in doc generation

Comment (mdedetrich) [reply project/plugins.sbt:39]: This was a big misunderstanding on my part, approving PR.

**Review by mdedetrich — APPROVED** [review]
Approving, need to double check if https://github.com/apache/incubator-pekko-connectors/pull/269 is required.

File: project/plugins.sbt
Comment (mdedetrich) [inline project/plugins.sbt:10]: @pjfanning Why is this here?

Comment (pjfanning) [reply project/plugins.sbt:10]: Some of the old sbt plugins have scala-xml v1 dependencies. pekko-grpc-sbt-plugin now brings in scala-xml v2 transitively. So I need to force sbt to use scala-xml v2 to stop it failing to run due to mismatched scala-xml versions.

**Comment by pjfanning** [comment]
> Approving, need to double check if #269 is required.

269 is not needed if this is merged

**Comment by mdedetrich** [comment]
> > Approving, need to double check if #269 is required.
> 
> 269 is not needed if this is merged

Go ahead and merge then

**Pull request merged**
